Binary_Search.py

```diff
@@ -0,0 +1,17 @@
+def binary_search(list,item):
```

Comment:
rename 'list'. we should follow the naming convention. and 'list' can be confused with List() declaration.